features/element-timing.yml

```diff
@@ -1,7 +1,7 @@
-draft_date: 2024-09-09
-name: Element Timing API
-description: TODO
+name: Element timing
+description: The `PerformanceElementTiming` interface contains render timing information for image and text node elements the developer annotated with an `elementtiming` attribute for observation.
 spec: https://wicg.github.io/element-timing/
+group: performance
 compat_features:
```

Comment:
There should probably be something like `html.global_attributes.elementtiming` in here, right?

Reply:
hm, there isn't such a key in BCD. The docs have no compat table either. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/elementtiming. Not sure if it is "global" either:

> This attribute may be applied to `<img>`, `<image>` elements inside an `<svg>`, poster images of `<video>` elements, elements which have a background-image, and elements containing text nodes, such as a `<p>`.

Reply:
It turns out there's no actual specification for this (though its reflection is via `Element`). I filed https://github.com/mdn/browser-compat-data/issues/24788